packages/simpleserialize.com/src/util/types.ts

```diff
@@ -1,16 +1,29 @@
 import {Type} from "@chainsafe/ssz";
-import {ssz} from "@chainsafe/lodestar-types";
+import {ssz} from "@lodestar/types";
 
-const {phase0, altair, bellatrix, allForks, ...primitive} = ssz;
+const {
+  phase0,
+  altair,
+  bellatrix,
+  capella,
+  deneb,
+  allForks,
+  allForksBlinded,
+  allForksBlobs,
+  allForksExecution,
+  allForksLightClient,
+  ...primitive
```

Comment:
would it make sense to export `primitive` as an object from `@lodestar/types`?

Reply:
may be both as devX would be affected, closing here for now

Reply:
Yeah was thinking both, not sure how useful exporting it as an object is besides using it here. I guess should not pollute the package API unless it is really required.